modules/developer-cli-odo-adding-storage-to-the-application-components.adoc

```diff
@@ -5,15 +5,67 @@
 [id="adding-storage-to-the-application-components_{context}"]
 = Adding storage to the application components
 
-Persistent storage keeps data available between restarts of {odo-title}. You can add storage to your components with the `odo storage` command.
+Persistent storage keeps data available between restarts of {odo-title}.
+Use `odo storage` to add data that must persist, such database files, dependencies, build artifacts (for example, a `.m2` directory for Maven applications), to your application.
 
 .Procedure
 
-* Add storage to your components:
+. Add the storage to your component:
 +
 [source,terminal]
 ----
-$ odo storage create nodestorage --path=/opt/app-root/src/storage/ --size=1Gi
+$ odo storage create <storage-name> --path=<path-to-the-directory> --size=<size>
 ----
 
-Your component now has 1 GB storage.
+. Push the storage to the cluser:
++
+[source,terminal]
+----
+$ odo push
+----
+
+. Verify that the storage is now attached to your component by listing all storage in the component:
++
+[source,terminal]
+----
+$ odo storage list
+----
++
+.Example output
++
+[source,terminal]
+----
+The component 'nodejs' has the following storage attached:
+NAME           SIZE     PATH
+mystorage      1Gi      /data     Pushed
+----
+
+. Delete the storage from your component:
++
+[source,terminal]
+----
+$ odo storage delete <storage-name>
+----
+
+. List all storage to verify that the storage state is `Locally Deleted`:
++
+[source,terminal]
+----
+$ odo storage list
+----
++
+.Example output
++
+[source,terminal]
+----
+The component 'nodejs' has the following storage attached:
+NAME           SIZE     PATH
+mystorage      1Gi      /data     Locally Deleted
```

Comment:
Should there be a `STATE` header in above `Locally Deleted` or does it not report that? I'm not sure.

Reply:
@bobfuru you're right! There i a header: https://github.com/openshift/odo/pull/2584
Updated.